src/ContentType/ContentTypeInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of sensiolabs-de/storyblok-bundle.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Bundle\ContentType;
+
+interface ContentTypeInterface
+{
+    /**
+     * @param array<string, array<string,mixed>&array{
+     *     content: array{
+     *         component: non-empty-string,
+     *     }
+     * }> $values
+     */
+    public function __construct(array $values);
```

Comment:
usually you don't put a __construct to an interface

Reply:
any better idea?

Reply:
Why is it important how I construct my DTO for the latter usage?

Reply:
To prevent service injections and or container injections. This is meant to be a DTO aka Entity in DDD language

Reply:
Never seen someone forcing the user this way to not do it, also Doctrine is not doing it for the entities.

Reply:
Do you have any better idea of enforcing this?

Reply:
I would just remove it, if someone tries to mess up, he will get what he asked for, right?

Reply:
There is no autowire in bundles as best practices, so there is no way/need to preent this.

And.... it's use role to not register as services their DTO.

And... in 7.3 there is an attribute to exclude from registration

Reply:
So shall we remove this? constructor definition?